graphx/src/main/scala/org/apache/spark/graphx/Graph.scala

```diff
@@ -105,6 +105,16 @@ abstract class Graph[VD: ClassTag, ED: ClassTag] protected () extends Serializab
   def checkpoint(): Unit
 
   /**
+   * Return whether this Graph has been checkpointed or not
+   */
+  def isCheckpointed: Boolean
```

Comment:
maybe beef up the documentation to say this only returns true if both vertices and edges are checkpointed?